ngraph/core/src/op/random_uniform.cpp

```diff
@@ -125,3 +126,69 @@ shared_ptr<Node> op::v8::RandomUniform::clone_with_new_inputs(const OutputVector
                                           m_global_seed,
                                           m_op_seed);
 }
+
+bool op::v8::RandomUniform::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const {
```

Comment:
Do we really need to have evaluate for this operation?

Reply:
We discussed this with @dmitry-gorokhov. In 2022.1 release they won't implement RandomUniform, so we need to have reference implementation in ngraph core. So we can already support models with RandomUniform operation. In next release when CPU implementation will be done reference implementation should be moved to interpreter backend.